pkg/util/sloconfig/colocation_config.go

```diff
@@ -68,6 +68,9 @@ func DefaultColocationStrategy() configuration.ColocationStrategy {
 		DegradeTimeMinutes:            pointer.Int64(15),
 		UpdateTimeThresholdSeconds:    pointer.Int64(300),
 		ResourceDiffThreshold:         pointer.Float64(0.1),
+		MidCPUThresholdPercent:        pointer.Int64(100),
+		MidMemoryThresholdPercent:     pointer.Int64(100),
+		MidUnallocatedPercent:         pointer.Int64(100),
```

Comment:
Currently the unallocated for Mid tier is still unstable. So I suggest setting the ratio to zero by default and enabling it when we know how to handle the risk.

Reply:
fixed.